Tools/TestWebKitAPI/Tests/WebKitCocoa/LoadAndDecodeImage.mm

```diff
@@ -312,4 +314,48 @@
     Util::run(&done);
 }
 
+TEST(WebKit, CreateIconDataFromImageDataSVGWithSubresource)
```

Comment:
Looks like `CGImageForProposedRect` don’t exist on iOS so we need to do something else, either make this (temporarily?) a Mac-only test or find a way to get at the pixels without using this Mac-only method.

Reply:
@annevk you can probably just use `UIImage.CGImage`: https://developer.apple.com/documentation/uikit/uiimage/cgimage?language=objc

Reply:
#43305 brings us a little closer, but I still need to understand why I get such a weird value on macOS. And unfortunately the iOS bot didn't print the description field for iOS as I was hoping.